src/XrdNet/XrdNetPMark.hh

```diff
@@ -41,11 +43,11 @@ class Handle
      {public:
 
       bool        getEA(int &ec, int &ac)
-                       {if (eCode >= 0) {ec = eCode; ac = aCode; return true;}
+                       {if (Valid()) {ec = eCode; ac = aCode; return true;}
                         ec = ac = 0; return false;
                        }
-
-      bool        Valid() {return eCode >= 0;}
+                  // According to the specifications, ExpID and actID can be equal to 0 for HTTP-TPC.
+      bool        Valid() {return (appName != nullptr && !strncmp("http-tpc",appName,8) && eCode == 0 && aCode == 0) || (eCode >= minExpID && eCode <= maxExpID && aCode >= minActID && aCode <= maxActID);}
```

Comment:
Please simplify this. The same rules apply to http as to xroot protocol. So, a basic simple test is all that is needed.

Reply:
I'm glad to read that. As this specification were written under the HTTP-TPC processing expectation, I thought it was only for `http-tpc`. Very good that I can simplify this now :)

Thanks for your review!